### DISCUSSION TITLE
Missing pytest

### DISCUSSION THREAD

**Discussion by mwsatman** [USC-ACTLab/crazyswarm#543] [Q&A]
Hi,

I am currently setting up crazyswarm on my ubuntu 20.04 setup with ROS noetic.
I followed the direction in [https://crazyswarm.readthedocs.io/en/latest/installation.html](url) in the Physical Robot and Simulation part.

I managed to get everything up until 
`$CSW_PYTHON -m pytest`
Then, I realized that the pytest.py is not available in my local.
I checked the crazyswarm github and it is not available as well.
Is this test deprecated?

**Comment by mwsatman** [comment]
Previously I received the error: no module named pytest
Which I originally think due to the missing of pytest.py
However, It seems I misunderstood the structure. There is no pytest.py.

I am still not sure what caused the error.
But once I delete the crazyswarm folder, and redo the instruction,
somehow, the pytest now works with 1 warning from the test_videoOutput.py

**Reply by jpreiss** [reply]
You should be installing `pytest` from `pip` in step 3 of the "physical robots and simulation" installation instructions.
